polkadot/node/network/collator-protocol/src/validator_side/tests/prospective_parachains.rs

```diff
@@ -327,6 +334,137 @@ async fn assert_persisted_validation_data(
 	}
 }
 
+// Combines dummy candidate creation, advertisement and fetching in a single call
```

Comment:
I don't think there's any reason to keep the prospective_parachains tests module anymore (as you removed the code for dealing with the old runtime. and for good reason). but would be good to do this in a follow-up PR. so that it's easier to review for now

Reply:
I agree but as you pointed out reviewing it would have been harder so I'll do it in a separate PR.